### ISSUE TITLE
Save Set description, status, and ordering

### ISSUE THREAD

**Issue by shengb** [frib-high-level-controls/save-set-restore#26]
A save set configuration could have name, description, status (active, or inactive), and time stamp.
It would be nice to show those information in the "Save Sets for xxx" window.

A practical solution at NSLS II is to show all active save set configuration on the top, and inactive one on the bottom.
For either active or inactive sets, they are ordered with reserved time ordering, latest on on the top, and oldest one on the bottom, like for example (no need to expose the configuration ID):
![screen shot 2016-03-16 at 9 37 21 am](https://cloud.githubusercontent.com/assets/3741779/13814135/80141780-eb5b-11e5-844d-b1b271835b81.png)

**Comment by berryma4** [comment]
We should talk to our users before doing this.
One, Config ids are a bad idea.
Two, a machine like FRIB will have more than just active/inactive.
Selecting configurations are done at the Experiment Service Description phase, in which the experimenters' needs are defined (and the physicists present their plan for the given experiment).
(How they will deliver the given isotope)

**Comment by shengb** [comment]
Exactly, I do not want to expose the config IDs here if you read my message above.
Others (name, description, time stamp, and status) are general enough.
Those are requirements captured at NSLS II.
Current implementation shows name only, which is not enough.

I agree that we should talk with our users. We can extend them to satisfy the requirements, for example more status rather than active/inactive.
But is there any specific reason not to implement those?

**Comment by berryma4** [comment]
Yes, the status is confusing.  In our last meeting our users said they would like to define the configuration hierarchy themselves.  (Some wanted to do folders based on experiment, others mentioned other folder hierarchies).

**Comment by shengb** [comment]
If I understand it properly, there are 2 different topics.
One it the view style (hierarchical view, or table view).
Another is the properties for each particular configuration (name, description, time stamp, status, and so on).
It would be great to support different views, with the property information.

However, as one of the users, the way I want to use is table view with those properties.
At NSLS II, the physicists/control room are pretty happy with this simple solution.

If other users want to use it on folder based, or folder hierarchies, and the app can support it, it would be great. We can have more than one instance deployed to satisfy different requirements.
